paydb_endpoint.py

```diff
@@ -302,3 +302,25 @@ def transaction_info():
     if tx.user != api_key.user and tx.recipient != api_key.user:
         return bad_request(web_utils.UNAUTHORIZED)
     return jsonify(dict(tx=tx.to_json()))
+
+@paydb.route('/user_update_email', methods=['GET', 'POST'])
+def user_update_email():
+    sig = request_get_signature()
+    content = request.get_json(force=True)
+    if content is None:
+        return bad_request(web_utils.INVALID_JSON)
+    params, err_response = get_json_params(content, ["api_key", "nonce", "email"])
+    if err_response:
+        return err_response
+    api_key, nonce, email = params
+    res, reason, api_key = check_auth(db.session, api_key, nonce, sig, request.data)
+    if not res:
+        return bad_request(reason)
+    ### using api_key.user to match to user
+    user = api_key.user
+    ### if user returns then update the email
+    if user:
```

Comment:
this line is not necessary (all api keys are associated with a user)

also if user was `None` the return statement would crash anyhow